app/root.tsx

```diff
@@ -25,9 +26,12 @@ export function Layout({ children }: { children: React.ReactNode }) {
         <Links />
       </head>
       <body>
```

Comment:
Why not create a file with the name super-admin in the app/routes ?

Please move this code to that file

Reply:
Sorry I don't understand

Reply:
Create a folder called data and inside it create a file called navigation. Move that array of navigation items to that file, export and import it into this file